lib/main.dart

```diff
@@ -1,173 +1,493 @@
 import 'package:flutter/material.dart';
+import 'package:autocomplete_textfield/autocomplete_textfield.dart';
+import 'package:flutter/services.dart';
 
 void main() => runApp(MyApp());
 
 class MyApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
+      title: "JBCH Interprep",
       home: Scaffold(
-        backgroundColor: Colors.grey[200],
-        body: SignUpScreen(),
-      ),
-    );
-  }
-}
-
-class SignUpScreen extends StatelessWidget {
-  @override
-  Widget build(BuildContext context) {
-    return Center(
-      child: SizedBox(
-        width: 400,
-        child: Card(
-          child: SignUpForm(),
+        appBar: AppBar(
+          title: Text("Interprep"),
+          centerTitle: true,
+          backgroundColor: Colors.brown[300],
+          actions: <Widget>[
+            IconButton(
```

Comment:
What is this button for?

Reply:
Its for the look :)

Reply:
Why not pass in null for `onPressed` if the button isn't supposed to do anything?